### PR TITLE
v2.2.2

### PR DESCRIPTION
**Pull request by cmarshak**
### Added
- Python 3.10 support with matrix in actions added back.

### Added
- Included Copernicus GLO-90 (as `glo_90`) and the missing GLO-30 tiles that are available as GLO-90 tiles as `glo_90_missing`
- Demonstration on how to fill in `glo-30` tiles that are missing with `glo-90` tiles.
- Exceptions that catch: a) no available tiles of specifed DEM as a `NoDEMCoverage` exception, b) badly specified `dem_name` as a `DEMNotSupported` exception and c) extent/bounds not of the form `xmin, ymin, xmax, ymax` as a `ValueError`
- API keyword argument `fill_in_glo_30` to fill in `glo_30` tiles that are missing, but whose corresponding `glo_90` tiles are not.
- Tests for Exceptions and added datasets

### Changed
- Since AWS registry removed zip, the `glo_30` and `glo_90` geojsons have precisely the tiles that are available (had to traverse bucket)
- Notebooks to organize data have been updated

### Fixed
- Fixed #33 i.e. missing `glo_30` rasters over Azerbaijan and Armenia by filling in with available `glo_90`.

### CODE REVIEW

**Comment by cmarshak** [comment]
I have two versions here since last release? How do I deal with that? Should I just update that to 2.2.1? @jhkennedy

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,28 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.2]
+
+### Added
+- Python 3.10 support with matrix in actions added back.
+
+## [2.2.1]
+
+### Added
```
Comment (jhkennedy) [inline CHANGELOG.md:16]:
```suggestion
```

**Review by jhkennedy — CHANGES_REQUESTED** [review]
Correct -- you'd want to consolidate them into one. We often do this as well and do a "changelog fix" PR right before releasing.

I also think this release might also qualify as a minor release, so I'd consider consolidating them to `2.3.0`.

**Comment by jhkennedy** [comment]
You could also have left this one open, and it would update once the changlog PR was merged in. (And edited the PR title)

**Pull request closed without merging**